pkg/eventcollector/event.go

```diff
@@ -0,0 +1,144 @@
+package eventcollector
+
+import (
+	"math/big"
+
+	log "github.com/golang/glog"
+	"github.com/pkg/errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/joincivil/civil-events-crawler/pkg/model"
+	"github.com/joincivil/go-common/pkg/eth"
+)
+
+type handleEventInputs struct {
+	event  *model.Event
+	errors chan<- error
+}
+
+// handleEvent is the func used for the goroutine pool that handles
+// incoming events fromt the watchers
+func (c *EventCollector) handleEvent(payload interface{}) interface{} {
+	inputs := payload.(handleEventInputs)
+	eventType := inputs.event.EventType() // Debug, remove later
+	hash := inputs.event.Hash()           // Debug, remove later
+	txHash := inputs.event.TxHash()       // Debug, remove later
+	log.Infof("handleEvent: handling event: %v, tx: %v, hsh: %v", eventType,
+		txHash.Hex(), hash) // Debug, remove later
```

Comment:
are these debugs still needed?

Reply:
probably not, but I like having these in here in case things go awry.  I prefer to just shut them off via the log level.

Reply:
probably not, but I like having these in here in case things go awry. I prefer to just shut them off via the log level.

Reply:
actually, i'll just rework how i trace this and remove these.